<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>4med::Главное окно</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Учёт</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Просмотр</string>
     </property>
     <addaction name="aShowDrugs"/>
     <addaction name="aShowRecipients"/>
     <addaction name="aShowDepartments"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Добавить</string>
     </property>
     <addaction name="aAddRegistration"/>
     <addaction name="aAddDrug"/>
     <addaction name="aAddRecipient"/>
     <addaction name="aAddDepartment"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aShowDrugs">
   <property name="text">
    <string>Лекарственные средства</string>
   </property>
  </action>
  <action name="aShowDepartments">
   <property name="text">
    <string>Отделы</string>
   </property>
  </action>
  <action name="aShowRecipients">
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </action>
  <action name="aAddRegistration">
   <property name="text">
    <string>Приход/расход</string>
   </property>
  </action>
  <action name="aAddDrug">
   <property name="text">
    <string>Лекарственное средство</string>
   </property>
  </action>
  <action name="aAddDepartment">
   <property name="text">
    <string>Отдел</string>
   </property>
  </action>
  <action name="aAddRecipient">
   <property name="text">
    <string>Сотрудник</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
